packages/react-cosmos-webpack/package.json

```diff
@@ -9,26 +9,22 @@
     "lib"
   ],
   "dependencies": {
-    "babel-loader": "^6.2.10",
-    "babel-register": "^6.16.3",
-    "css-loader": "^0.28.1",
-    "express": "^4.14.0",
-    "fs-extra": "^3.0.1",
+    "babel-register": "^6.26.0",
```

Comment:
It's used [here](https://github.com/react-cosmos/react-cosmos/blob/c84ea3308f3a8b41df995b5189b1fb22ff0b27f1/packages/react-cosmos-webpack/bin/cosmos.js#L7). The point is to enable Babel on the server and allow users to write `export default {` instead of `module.exports ={` in cosmos.config.js. Not sure if it's worth it either, but a different story.